rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -96,15 +94,15 @@ type OptionPayout extends PayoutBase: <" The option payout specification terms.
     observationTerms ObservationTerms (0..1) <"Class containing terms that are associated with observing a price/benchmark/index across either single or multple observations. To be used for option contracts that reference a benchmark price.">
     schedule CalculationSchedule (0..1) <"Allows the full representation of a payout by defining a set of schedule periods. It supports standard schedule customization by expressing all the dates, quantities, and pricing data in a non-parametric way.">
     delivery AssetDeliveryInformation (0..1) <"Contains the information relative to the delivery of the asset.">
-    underlier Product (1..1) <"The product underlying the option, which can be of any type including ContractualProduct or Security.">
+    productUnderlier ProductUnderlier (1..1) <"The financial product underlying the option, which can be of any type including an Asset, Basket, Index or a NonTransferableProduct.">
```

Comment:
Remove `Observable` from `ObservationTerms`, otherwise it would allow 2 different paths to define `Observable`. Same comment applies to `PerformancePayout`.